spec/frequency_classes_service_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'frequency_classes_service'
+
+describe 'FrequencyClassesService' do
+  let(:frequency_classes_service) { FrequencyClassesService.new }
+  let(:sample_word) { 'haben' }
+
+  describe '#initialize' do
+    it 'should return a FrequencyClassesService instance' do
```

Comment:
(if it helps you debug when something fails, such as when the class name changes etc. then keep it, but I suspect any other related tests will provide the same error messages regardless in this particular case)